freud/locality.pyx

```diff
@@ -183,6 +182,8 @@ cdef class NeighborQueryResult:
 
     def __iter__(self):
         cdef freud._locality.NeighborBond npoint
+        cdef freud._locality.NeighborBond ITERATOR_TERMINATOR
+        ITERATOR_TERMINATOR = freud._locality.NeighborBond(-1, -1, 0.0, 1.0)
```

Comment:
I couldn't figure out how to export the static class member and import it in Cython. Windows rules around linking and exported symbols make this really tough. This is probably too complex for us to worry about, since the workaround is simple and it's only needed in one place in the Cython code.